src/v2/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx

```diff
@@ -16,10 +16,19 @@ export interface ArtworkImageBrowserProps {
 export const ArtworkImageBrowser: React.FC<ArtworkImageBrowserProps> = ({
   artwork,
 }) => {
-  const { images } = artwork
+  const { images, video } = artwork
+
+  let length = 0
+  if (images?.length) {
+    length += images.length
+  }
+
+  if (video) {
+    length += 1
+  }
```

Comment:
Avoid this `let` it's really confusing to try to read this.

```ts
const length = images.length + video ? 1 : 0
```

Reply:
All this stuff seems like it'd be a lot clearer if we modeled with the expectation of multiple videos. In which case we'd have an empty videos array. Or even simpler, a union of `figures`